Model/ResourceModel/Customer.php

```diff
@@ -50,6 +50,6 @@ class Customer extends AbstractDb
      */
     public function _construct()
     {
-        $this->_init(self::TABLE_NAME, CustomerInterface::CUSTOMER_ID);
+        $this->_init(self::TABLE_NAME, CustomerInterface::ID);
```

Comment:
That's the new table primary key. CustomerID should have magento customer ID and not some random autogenerated number like it was before